app.py

```diff
@@ -0,0 +1,81 @@
+"""Application for managing tarantool repositories."""
```

Comment:
BTW, I vote for using mkrepo from pypi, not as a submodule.

Reply:
Ok.